jerry-core/jerry-port.h

```diff
@@ -139,6 +140,14 @@ bool jerry_port_get_time_zone (jerry_time_zone_t *tz_p);
  */
 double jerry_port_get_current_time (void);
 
+#ifndef CONFIG_DISABLE_ES2015_PROMISE_BUILTIN
+
+typedef uint32_t (*jerry_job_handler_t) (void *);
+
+void jerry_port_jobqueue_enqueue (jerry_job_handler_t handler, void *job_p);
```

Comment:
I think it is too early to set the seperate `CONFIG_DISABLE_ES2015_JOBQUEUE`, the reasons are as follows:
* for now, only promise will use jobqueue. Enabling only one of "JOBQUEUE" and "PROMISE" does not make much sense. 
* in the future, when we plan to support `module` (which also use jobqueue), we will enable jobqueue by `#if !defined(CONFIG_DISABLE_PROMISE) || !defined(CONFIG_DISABLE_MODULE)`
* will `setTimeout` use the jobqueue? it is out of jerry-core's scope, because setTimeout is a external api. The binding code can directly use the jobqueue in the port no matter whether job queue is enabled in jerry-core. (That is another reason why I think the loop should not be in jerry-core)